src/main/java/io/github/oliviercailloux/y2018/jconfs/ConferencesFromICal.java

```diff
@@ -2,83 +2,56 @@
 
 import java.io.File;
 import java.io.FileInputStream;
+import java.io.FileReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
-import java.net.URI;
-import java.net.URISyntaxException;
 import java.net.URL;
 import java.text.ParseException;
 import java.time.LocalDate;
-
 import java.util.LinkedHashSet;
 import java.util.Set;
+import com.google.common.base.Preconditions;
 
 import net.fortuna.ical4j.data.ParserException;
-import net.fortuna.ical4j.model.DateTime;
 
 public class ConferencesFromICal implements ConferencesRetriever {
 
-	/**
-	 * A constructor to initialize a ConferencesFromICal
-	 * 	
-	 * @throws URISyntaxException
-	 */
-	public ConferencesFromICal() {
-	}
-
 	@Override
-	public Set<Conference> retrive() throws IOException, NumberFormatException, ParserException, ParseException {
-
+	public Set<Conference> retrieve() throws IOException, NumberFormatException, ParserException, ParseException {
 		String filePath = ConferencesFromICal.class.getClassLoader().getResource("icaldata").getFile();
```

Comment:
Vous supposez que vos conférences lues sont dans votre class path ? Pourquoi ? Je ne suis pas sûr de ce que vous voulez faire.

Reply:
il s'agit là d'une méthode réalisée par l'ancien groupe et que nous envisageons pas d'utiliser pour le moment. Je pense que lire les conférences dans un classpath est plus pratique dans un premier temps si l'on s'occupe d'abord de l'aspect code (stockage de la conférence, application des bonnes méthodes pour la recherche etc). il suffira par la suite de seulement modifier le filepath, le reste ne changera pas.

Reply:
Après concertation avec les membres de l'équipe la méthode sera surement retirée lors de la prochaine itération.

Reply:
Après concertation avec les membres de l'équipe la méthode sera surement retirée lors de la prochaine itération.